lemminx-liberty/src/test/java/io/openliberty/LibertyLemminxTestsUtils.java

```diff
@@ -0,0 +1,42 @@
+package io.openliberty;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.util.List;
+
+import org.eclipse.lemminx.commons.TextDocument;
+import org.eclipse.lemminx.dom.DOMDocument;
+import org.eclipse.lemminx.services.XMLLanguageService;
+import org.eclipse.lsp4j.CompletionItem;
+import org.eclipse.lsp4j.TextDocumentItem;
+
+/**
+ * Utils class for Liberty Lemminx Test Cases
+ * 
+ * adapted from:
+ * https://github.com/eclipse/lemminx-maven/blob/master/lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/MavenLemminxTestsUtils.java
+ */
+public interface LibertyLemminxTestsUtils {
+
+    public static DOMDocument createDOMDocument(String resourcePath, XMLLanguageService languageService)
```

Comment:
Maybe rename to `createServerXMLDOMDocument` since the dom document that is returned will always be named server.xml